authors/apps/articles/models.py

```diff
@@ -239,3 +239,19 @@ def _update_article_tags(self, instance, new_tag_list):
             # Add the tag to the article instance.
             instance.tags.add(the_tag)
         return True
+
+
+class Bookmark(models.Model):
+    """Store bookmarks"""
```

Comment:
@ElMonstro, this docstring is not descriptive enough.

Reply:
Will fix this